### DISCUSSION TITLE
/run/user/0/pulse/pid or /run/user/1000/pulse/pid

### DISCUSSION THREAD

**Discussion by mayor686** [quickemu-project/quickemu#1525] [Quickemu]
Hi, i have the following error on debian sid:
"qemu-system-x86_64: could not stat pidfile /run/user/0/pulse/pid: No such file or directory"

in the folder /run/user/0/ there isn't any pulse/pid file; 

it is inside /run/user/1000/pulse/pid.

I tried to create a symlink but i have the following error:

"Failed to create secure directory (/run/user/0/pulse): Too many levels of symbolic links"

Anyone can help me? It is a bug?
Thanks

**Comment by lj3954** [comment]
Are you running quickemu as root? That is generally not a good idea.
